app/views/tutorial/a9.html

```diff
@@ -56,6 +66,80 @@ <h3 class="panel-title">How Do I Prevent It?</h3>
             </div>
         </div>
 
+        <div id="source-code-example" class="panel panel-info">
+            <div class="panel-heading">
+                <h3 class="panel-title">Insecure Dependencies Example</h3>
+            </div>
+            <div class="panel-body">
+
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">Description</h3>
+                    </div>
+                    <div class="panel-body">
+
+                        <p>
+                            The demo web application is using a popular library called <a href="https://github.com/chjj/marked"> Marked </a> which is a Markdown parser in JavaScript and provides an easy way to integrate markdown syntax for rich text to a website, replacing the need to build WYSIWYG editors.
+                        </p>
+
+                        <p>
+                            This library is so popular that it has been downloaded almost <strong> 3 million </strong> times in the last month and received more than <strong> 11,000 </strong> stars on GitHub.
```

Comment:
yeah, would be a bit difficult to predict the future but I tried to make it generic, take a look now.